x/lockup/spec/05_keeper.md

```diff
@@ -0,0 +1,36 @@
+<!--
+order: 5
+-->
+
+# Keepers
+
+## Lock Keeper
+
+Lock Keeper provide utility functions to query user's locks.
+
+```go
+// Keeper is the interface for lockup module keeper
+type Keeper interface {
+	// Return full balance of the module
+	GetModuleBalance(sdk.Context) sdk.Coins
+	// Return locked balance of the module
+	GetModuleLockedAmount(sdk.Context) sdk.Coins
+
+	// Returns whole balance deposited by the user which is not withdrawn yet
+	GetAccountPoolBalance(sdk.Context, sdk.AccAddress) sdk.Coins
```

Comment:
GetAccountUnlockableBalance

Reply:
Instead of `GetAccountUnlockableBalance` used `GetAccountUnlockableCoins` since we use `Coins` keyword for every other thing.